builtin/providers/aws/resource_aws_iam_group.go

```diff
@@ -93,6 +92,26 @@ func resourceAwsIamGroupReadResult(d *schema.ResourceData, group *iam.Group) err
 	return nil
 }
 
+func resourceAwsIamGroupUpdate(d *schema.ResourceData, meta interface{}) error {
+	if d.HasChange("name") || d.HasChange("path") {
+		iamconn := meta.(*AWSClient).iamconn
+		on, nn := d.GetChange("name")
+		op, np := d.GetChange("path")
+		fmt.Println(on, nn, op, np)
```

Comment:
Is this for debugging? We should probably output the `iam.UpdateGroupInput` in a `[DEBUG]` log statement, similar to other areas: 

We don't seem to use `op` either, so we can just use `_` here unless you plan on logging it in the before mentioned `DEBUG` 

- https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/resource_aws_instance.go#L354